consensus/pacemaker/pacemaker.go

```diff
@@ -0,0 +1,280 @@
+package pacemaker
```

Comment:
In the future use `git mv` rather than deleting and creating a new file. It makes it easier to understand which parts of the file were changed. Going to trust the business logic wasn't modified other than where necessary.

e.g `git mv consensus/pacemaker.go consensus/pacemaker/pacemaker.go`

----

You might even be able to do it in this PR.

1. Copy-paste this file to not lose the changes.
2. Run these commands
```
git checkout consensus/pacemaker.go main
git rm consenus/pacemaker/pacemaker.go
git commit -am "..."
git mv consensus/pacemaker.go consensus/pacemaker/pacemaker.go
```
3. Copy paste your changes back.
```
git commit -am "..."
git push
```

----

Are you open to giving this a shot in this PR? It'll make it easier to review/validate the changes versus the file move.

Reply:
Done, thanks for the information.

Reply:
Did you manage to do this? I'm still seeing the entire file as new.

Reply:
Re-did following commands:


1. `git checkout main consensus/pacemaker.go`
2. `git rm consensus/pacemaker/pacemaker.go`
3. `git commit -am "remove pacemaker"`
4. ` git push origin head` (I pushed to show the intermediate step performed)
5. pasted previously copied pacemaker.go
6. `git mv consensus/pacemaker.go consensus/pacemaker/pacemaker.go`
7. `git commit -am "add updated pacemaker"`
8. `git push origin head`

Reply:
Seems like github isn't showing the changes, but I made a commit myself so it's easier to review: [`aacef7e` (#427)](https://github.com/pokt-network/pocket/pull/427/commits/aacef7e3ca25cc648150c37e3dcad3f8e791beac)